script/Ciel/Day 13/13_01_02_CIEL13_7C.txt

```diff
@@ -11,7 +11,7 @@ I'm the one you're really after, right, Noel?"
 [sha:99b90bc4120fe60c2ec33be3ba4a07d1a087fd89]{
 -- Page 10011, Offset 36098.
 -- 「……！」
-"...!"
+"!"
```

Comment:
Weird why this is just an exclamation mark in the JP.
>Ahah!

Heck maybe even add a heart.